libmac/mac.c

```diff
@@ -1563,3 +1563,14 @@ uint64_t kdebug_trace_string(uint32_t debugid, uint64_t str_id,
                              const char *str) {
   return 0;
 }
+
+void __darwin__ZNSt3__15mutex4lockEv() {
```

Comment:
Any reason you used inline assembly instead of the first argument? I'm guessing

```
void __darwin__ZNSt3__15mutex4lockEv(struct __darwin_pthread_mutex_t *t) {
  LOGF("std::mutex::lock: sig=%lx", t->sig);
  __darwin_pthread_mutex_lock(t);
}
```

would work.